templates/default/net/netadd.html

```diff
@@ -35,7 +35,7 @@ <H1>{$layout.pagetitle}</H1>
 			<input type="text" size="23" name="netadd[address]" value="{$netadd.address}" required
 				{tip text="Enter IP address and select mask" trigger="address"}>
 			/
-			<select size="1" name="netadd[prefix]">
+			<select size="1" name="netadd[prefix]" {tip text="{trans('Choose prefix')}"}>
 				{section name=prefixlist loop=$prefixlist.id}
```

Comment:
Z tego, co widzę, to {tip text=...} robi translatę tekstu.

Reply:
Uproszczenie zastosowane.